applications/StructuralMechanicsApplication/tests/periodic_test/periodic_linear_test_materials.json

```diff
@@ -0,0 +1,17 @@
+{
+    "properties" : [{
+        "model_part_name" : "Parts_solid",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw"
```

Comment:
please remove `KratosMultiphysics.StructuralMechanicsApplication.`, it is not needed and only generates warnings